spec/injection_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'injection'
+
+describe Array do
+
+  describe 'injection' do
+
+	  it { is_expected.to respond_to :injection }
+
+    context 'with a block' do
+
+      it 'can add numbers together' do
+        expect([1, 2, 3, 4].injection { |acc, el| acc + el }).to eq 10
+      end
+
+      it 'can multiply numbers together' do
+        expect([1, 2, 3, 4].injection { |acc, el| ac * el }).to eq 24
```

Comment:
Unused block argument - `acc`. If it's necessary, use `_` or `_acc` as an argument name to indicate that it won't be used.